src/Symfony/Component/Serializer/Tests/DependencyInjection/SerializerPassTest.php

```diff
@@ -83,17 +83,20 @@ public function testServicesAreOrderedAccordingToPriority()
 
     public function testBindSerializerDefaultContext()
     {
+        $context = ['enable_max_depth' => true];
+
         $container = new ContainerBuilder();
         $container->setParameter('kernel.debug', false);
-        $container->register('serializer')->setArguments([null, null]);
+        $container->register('serializer')->setArguments([null, null, []]);
```

Comment:
```suggestion
        $container->register('serializer')->setArguments([null, null, null]);
```

Reply:
as the actual argument requires an array, this is correct. What's wrong is to pass `null` to both previous arguments which expects arrays as well. We may want to fix this or not, I wouldn't care much tbh